actualbudget/docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: actualbudget_web_1
+      APP_PORT: 5006
+
+  web:
+    image: actualbudget/actual-server:23.5.0@sha256:e2e6daa61c9049e2a7b637500f18a1a2c4ddee9af4ac4b1d90d1f2f194383b47
```

Comment:
Looks like a new update has come out since your submission, so we can bump! https://github.com/actualbudget/actual/releases/tag/v23.6.0